assets/dev/scss/frontend/widgets/tabs.scss

```diff
@@ -64,6 +64,35 @@
 		}
 	}
 
+	&.elementor-tabs-alignment- {
+
+		&center {
+
+			.elementor-tabs-wrapper {
+				display: flex;
```

Comment:
`.elementor-tabs-wrapper` can always be in `display: flex`, then you will only need to write it once

Reply:
Fix: https://github.com/elementor/elementor/pull/12002/commits/b641bb3a5885ed4da6dd63ff79138ef28aa10d9f